### PR TITLE
Add cafe.avery.Delfin

### PR DESCRIPTION
**Pull request by coolavery**
### Please confirm your submission meets all the criteria

- [x] I have read the [App Requirements][reqs] and [App Maintenance][maint] pages.
- [x] My pull request follows the instructions at [App Submission][submission].
- [x] I have [built][build] and tested the submission locally.
- [x] I am using only the minimal set of permissions. *(If not, please explain each non-standard permission.)*
- [x] All assets referenced in the manifest are redistributable by any party.  If not, the unredistributable parts are using an extra-data source type.
- [x] I am an upstream contributor to the project. If not, I contacted upstream developers about submitting their software to Flathub. **Link:**
- [x] I own the domain used in the application ID or the domain has a policy for delegating subdomains (e.g. GitHub, SourceForge).
- [x] Any additional patches or files have been submitted to the upstream projects concerned. *(If not, explain why.)*

[reqs]: https://docs.flathub.org/docs/for-app-authors/requirements
[maint]: https://docs.flathub.org/docs/for-app-authors/maintanance
[submission]: https://docs.flathub.org/docs/for-app-authors/submission
[build]: https://docs.flatpak.org/en/latest/first-build.html

### CODE REVIEW

**Comment by coolavery** [comment]
bot, build cafe.avery.Delfin

File: cafe.avery.Delfin.yaml [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+  append-path: "/usr/lib/sdk/rust-stable/bin:/usr/lib/sdk/llvm16/bin"
+  env:
+    CARGO_REGISTRIES_CRATES_IO_PROTOCOL: sparse
+    CARGO_TARGET_X86_64_UNKNOWN_LINUX_GNU_LINKER: clang
+    CARGO_TARGET_X86_64_UNKNOWN_LINUX_GNU_RUSTFLAGS: "-C link-arg=-fuse-ld=/usr/lib/sdk/rust-stable/bin/mold"
+    CARGO_TARGET_AARCH64_UNKNOWN_LINUX_GNU_LINKER: clang
+    CARGO_TARGET_AARCH64_UNKNOWN_LINUX_GNU_RUSTFLAGS: "-C link-arg=-fuse-ld=/usr/lib/sdk/rust-stable/bin/mold"
+  test-args:
+    - "--socket=x11"
+    - "--share=network"
```
Comment (hfiguiere) [inline cafe.avery.Delfin.yaml:33]: this is not needed. it's never used and can't be.
```suggestion
```

**Comment by coolavery** [comment]
bot, build cafe.avery.Delfin

File: cafe.avery.Delfin.yaml [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+
+build-options:
+  append-path: "/usr/lib/sdk/rust-stable/bin:/usr/lib/sdk/llvm16/bin"
+  env:
+    CARGO_REGISTRIES_CRATES_IO_PROTOCOL: sparse
+    CARGO_TARGET_X86_64_UNKNOWN_LINUX_GNU_LINKER: clang
+    CARGO_TARGET_X86_64_UNKNOWN_LINUX_GNU_RUSTFLAGS: "-C link-arg=-fuse-ld=/usr/lib/sdk/rust-stable/bin/mold"
+    CARGO_TARGET_AARCH64_UNKNOWN_LINUX_GNU_LINKER: clang
+    CARGO_TARGET_AARCH64_UNKNOWN_LINUX_GNU_RUSTFLAGS: "-C link-arg=-fuse-ld=/usr/lib/sdk/rust-stable/bin/mold"
+
```
Comment (bbhtt) [inline cafe.avery.Delfin.yaml:31]: Cleanup: `"*.a"`

**Comment by coolavery** [comment]
bot, build cafe.avery.Delfin

**Comment by coolavery** [comment]
Thanks for the reviews all! 

> Cool stuff. The only thing I'm missing is subtitles support, but I'm saying that as a user, not a reviewer.

@barthalion This might be due to this version not supporting subtitles that aren't baked into the video stream, which will be fixed in the next release

**Pull request closed without merging**